middleware.ts

```diff
@@ -8,36 +8,59 @@ import { LOGIN_URL } from './constants/network';
 const handleI18nRouting = createMiddleware({
   // A list of all locales that are supported
   locales: ['en', 'ko'],
-
   // If this locale is matched, pathnames work without a prefix (e.g. `/about`)
   defaultLocale: 'ko',
-
   localePrefix: 'as-needed',
-
   localeDetection: !AUDIT_LOCALE_COOKIE_DISABLED,
 });
 
 const isAuthRequired = (pathname: string) => {
   if (pathname.startsWith('/en')) pathname = pathname.slice(3);
-
-  if (pathname.startsWith('/admin')) return true;
-
-  return pathname.endsWith('create') || pathname.endsWith('edit');
+  return pathname.startsWith('/admin') || pathname.endsWith('create') || pathname.endsWith('edit');
 };
 
+const generateCSPHeader = (nonce: string) =>
```

Comment:
이거 카카오맵 가져오는 것도 허용해줘야 할듯...!
```
Refused to load the script 'https://dapi.kakao.com/v2/maps/sdk.js?appkey=...&autoload=false' 
because it violates the following Content Security Policy directive: "script-src 'self' 'unsafe-eval' 'unsafe-inline'". 
Note that 'script-src-elem' was not explicitly set, so 'script-src' is used as a fallback.
```